configstore/tests/test_zintegration.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+import configstore
+
+DOTENV_CONTENTS = u'''
+SECRET_KEY=1234dot
+ENVIRONMENT=PRODUCTION
+'''
+
+DOTENV_EMPTY = u''''''
+
+
+def test_env_var_dotenv_success(monkeypatch, tmpdir):
+    monkeypatch.setenv('ENVIRONMENT', 'STAGING')
+    p = tmpdir.join("config.env")
```

Comment:
`p` -> `dotenv_file`

J'suis désolé, faut pas le prendre mal. C'est juste un bon exemple que j'ai relus trois fois avant de catcher c'était quoi `p` au lieu de le lire une seule fois. :)

Reply:
J’avoue que le nom ne rend pas explicite le type de l’objet
(p = path = chemin?  ou c’est un objet fichier? faut-il le fermer?)

Reply:
En fait avec pytest plus récent on peut utiliser pathlib de la bibli standard!
https://docs.pytest.org/en/latest/tmpdir.html#the-tmp-path-fixture